lib/shopwareStoreCommander.js

```diff
@@ -176,35 +190,22 @@ module.exports = class ShopwareStoreCommander {
 
             if (review.status.id !== 1) {
```

Comment:
This condition (and possibly the one in the `else if` below) should go into the while condition part.